demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1255,3 +1253,72 @@ def name_not_contain_the_type(self):
 
         return True
+
+    def is_valid_endpoint_command(self):
+        """
+        Check if the endpoint command in yml is valid by standard.
+        This command is separated than other reputation command as the inputs are different standard.
+
+        Returns:
+            true if the inputs and outputs are valid.
+        """
+        commands = self.current_file.get('script', {}).get('commands', [])
+
+        if 'endpoint' not in [x.get('name') for x in commands]:
+            return True
+        else:
+            # extracting the specific command from commands.
+            endpoint_command = [arg for arg in commands if arg.get('name') == 'endpoint'][0]
+            return self._is_valid_endpoint_inputs(endpoint_command, required_arguments=ENDPOINT_FLEXIBLE_REQUIRED_ARGS) \
+                and self._is_valid_endpoint_outputs(endpoint_command)
+
+    def _is_valid_endpoint_inputs(self, command_data, required_arguments):
+        """
+        Check if the input for endpoint commands includes at least one required_arguments,
+        and that only ip is the default argument.
+        Returns:
+            true if the inputs are valid.
+        """
+        endpoint_command_inputs = command_data.get('arguments', [])
+        existing_arguments = {arg['name'] for arg in endpoint_command_inputs}
+
+        # checking at least one of the required argument is found as argument
+        if not set(required_arguments).intersection(existing_arguments):
+            error_message, error_code = Errors.reputation_missing_argument(list(required_arguments),
+                                                                           command_data.get('name'),
+                                                                           all=False)
+            if self.handle_error(error_message, error_code, file_path=self.file_path):
+                self.is_valid = False
+                return False
+
+        # checking no other arguments are default argument:
+        default_args_found = [(arg.get('name'), arg.get('default', False)) for arg in endpoint_command_inputs]
+        command_default_arg_map = BANG_COMMAND_ARGS_MAPPING_DICT[command_data.get('name')]
+        default_arg_name = command_default_arg_map['default']
+        other_default_args_found = list(filter(
+            lambda x: x[1] is True and x[0] not in default_arg_name, default_args_found))
+        if other_default_args_found:
+            error_message, error_code = Errors.wrong_default_argument(default_arg_name, command_data.get('name'))
+            if self.handle_error(error_message, error_code, file_path=self.file_path):
+                self.is_valid = False
+                return False
+        return True
+
+    def _is_valid_endpoint_outputs(self, command_data):
+        context_standard = "https://xsoar.pan.dev/docs/integrations/context-standards"
+        output_for_reputation_valid = True
+        command_name = command_data.get('name')
```

Comment:
no need, we already know it is `endpoint`, you can use the const from constants.py